pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -1590,38 +1590,42 @@ func TestEBSVolumeCountConflicts(t *testing.T) {
 			},
 		},
 	}
-	ebsPVCPod := &v1.Pod{
-		Spec: v1.PodSpec{
-			Volumes: []v1.Volume{
-				{
-					VolumeSource: v1.VolumeSource{
-						PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
-							ClaimName: "someEBSVol",
+	onePVCPod := func(filterName string) *v1.Pod {
```

Comment:
Can we refactor this higher level function so that it could be reused?

Reply:
done